### PR TITLE
MULE-8252 When the status code is set, http listener should auto comp…

### PR DESCRIPTION
**Pull request by mtqp**
MULE 8252 When the status code is set, http listener should auto complete de reason phrase

### CODE REVIEW

File: modules/http/src/main/java/org/mule/module/http/api/HttpConstants.java [hunk 1]
```diff
@@ -6,6 +6,10 @@
  */
 package org.mule.module.http.api;
 
+import com.google.common.collect.Collections2;
```
Comment (afelisatti) [inline modules/http/src/main/java/org/mule/module/http/api/HttpConstants.java]: Remove import, it's not used.

File: modules/http/src/main/java/org/mule/module/http/api/HttpConstants.java [hunk 2]
```diff
@@ -6,6 +6,10 @@
...
+
+import java.util.Arrays;
```
Comment (afelisatti) [inline modules/http/src/main/java/org/mule/module/http/api/HttpConstants.java]: Same as above.

File: modules/http/src/main/java/org/mule/module/http/api/HttpConstants.java (hunk 1 above)
Comment (mtqp) [inline modules/http/src/main/java/org/mule/module/http/api/HttpConstants.java]: Done

File: modules/http/src/main/java/org/mule/module/http/api/HttpConstants.java (hunk 2 above)
Comment (mtqp) [inline modules/http/src/main/java/org/mule/module/http/api/HttpConstants.java]: Done

**Pull request merged**